packages/tasks/package.json

```diff
@@ -25,6 +25,9 @@
 		"format:check": "prettier --check .",
 		"prepublishOnly": "pnpm run inference-codegen && git diff --name-only --exit-code src && pnpm run build",
 		"build": "tsup src/index.ts --format cjs,esm --clean && tsc --emitDeclarationOnly --declaration",
+		"watch:export": "tsup src/index.ts --format cjs,esm --watch",
+		"watch:types": "tsc --emitDeclarationOnly --declaration --watch",
+		"watch": "npm-run-all --parallel watch:export watch:types",
```

Comment:
parallel is fine, or should we first tsup, then tsc (i.e. sequentially)?

(i don't know enough to know)

Reply:
the process won't exit in watch mode, so only the 1st process will run in sequential mode.